auslib/blobs/schemas/superblob.yml

```diff
@@ -1,31 +1,67 @@
 ---
```

Comment:
Pretty good work on this schema. There's an issue on line 8 with indentation, but after I fixed that it worked fine. I'm surprised that the tests didn't pick up the error with the schema...